src/zenml/integrations/mlflow/flavors/mlflow_experiment_tracker_flavor.py

```diff
@@ -15,21 +15,24 @@
 
 from typing import TYPE_CHECKING, Any, Dict, Optional, Type
 
-from pydantic import root_validator
+from pydantic import model_validator
 
 from zenml.config.base_settings import BaseSettings
 from zenml.experiment_trackers.base_experiment_tracker import (
     BaseExperimentTrackerConfig,
     BaseExperimentTrackerFlavor,
 )
 from zenml.integrations.mlflow import MLFLOW_MODEL_EXPERIMENT_TRACKER_FLAVOR
+from zenml.logger import get_logger
 from zenml.utils.secret_utils import SecretField
 
 if TYPE_CHECKING:
     from zenml.integrations.mlflow.experiment_trackers import (
         MLFlowExperimentTracker,
     )
 
+logger = get_logger(__name__)
```

Comment:
Where do you use logger?

Reply:
Oh, good catch! I was using it previously but ended up removing it. Cleaned this up now.